chart/values.yaml

```diff
@@ -12,7 +12,7 @@ replicaCount: 1
 
 image:
   repository: datainfrahq/druid-operator
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
Why always?

Reply:
i think, i did this for my development purposes where i killed the container to "reload" the operator with new code

Reply:
reverted